.projenrc.js

```diff
@@ -42,34 +38,7 @@ const project = new cdk.JsiiProject({
       npmDistTag: 'latest-1',
     },
   },
-
-  minNodeVersion: '14.17.0',
-
-  // jsii configuration
-  publishToMaven: {
-    javaPackage: 'org.cdk8s',
-    mavenGroupId: 'org.cdk8s',
-    mavenArtifactId: 'cdk8s',
-  },
-  publishToPypi: {
-    distName: 'cdk8s',
-    module: 'cdk8s',
-  },
-  publishToNuget: {
-    dotNetNamespace: 'Org.Cdk8s',
-    packageId: 'Org.Cdk8s',
-  },
-  publishToGo: {
-    gitUserName: 'cdk8s-automation',
-    gitUserEmail: 'cdk8s-team@amazon.com',
-    gitBranch: '1.x',
-    moduleName: 'github.com/cdk8s-team/cdk8s-core-go',
-  },
-  autoApproveOptions: {
-    allowedUsernames: ['cdk8s-automation'],
-    secret: 'GITHUB_TOKEN',
-  },
-  autoApproveUpgrades: true,
+  golangBranch: '1.x',
```

Comment:
This is why it wasn't implemented as a regular backport of https://github.com/cdk8s-team/cdk8s-core/pull/1068